app/models/group/sektion.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2012-2021, Schweizerischer Kanu-Verband. This file is part of
+#  hitobito_skv and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_skv.
+
+
+class Group::Sektion < ::Group
+  class JsCoach < ::Role
+    self.permissions = [:group_full]
+  end
+
+  class Junior < ::Role
+    self.permissions = [:group_read]
+  end
+
+  class Aktivmitglied < ::Role
+    self.permissions = [:group_read]
+  end
+
+  class Passivmitglied < ::Role
+    self.permissions = [:group_read]
+  end
+
+  class Ehrenmitglied < ::Role; end
+
+  roles JsCoach, Junior, Aktivmitglied, Passivmitglied, Ehrenmitglied
+
+  children(Group::Projektgruppe, Group::SektionVorstand)
```

Comment:
Ich würde `children` lieber oben in der Gruppenklasse sehen. Kannst du das noch hochschieben?

Ausserdem sollte der Vorstand glaube ich immer angelegt werden (`default_children [Group::SektionVorstand]`)